.github/workflows/main.yml

```diff
@@ -21,6 +21,8 @@ jobs:
       # Disabled for now, causing issues
       # - run: flutter test
       - run: flutter build apk
+<<<<<<< HEAD
+=======
```

Comment:
fix this conflict. leave it as master and do not change this file

Reply:
updated origin master